README.md

```diff
@@ -1 +1,12 @@
-# lc-3-vm
\ No newline at end of file
+# VM that uses LC-3 architecture
+
+Following [this tutorial.](https://www.jmeiners.com/lc3-vm/)
+
+## OP Codes
+
+
+### How to run
+- Clone the repository with `git clone`.
+- Enter the directory containing the VM files and run `make run`.
```

Comment:
I would clarify here that `make run` doesn't work yet. And then when implemented remove the clarification.

Reply:
Done!